testdata/testcases/decision_task_tests.yml

```diff
@@ -69,11 +69,15 @@ testSuite:
             name: public/docker-contexts
             path: artifact1
             type: directory
+          - expires: '0001-01-01T00:00:00.000Z'
+            name: public/dockerImage.tar.gz
+            path: image.tar.gz
+            type: file
         command:
           - - bash
             - '-cx'
             - >-
-              podman run --name taskcontainer --privileged --cap-add=SYS_PTRACE -v
+              podman run --name taskcontainer --privileged -v
```

Comment:
`--privileged` and  `--cap-add=SYS_PTRACE` cannot be used together in a real test. The `podman run` command will fail out saying you cannot have them both, so I removed the `allowPtrace` feature for now so the test creates a working command.